k8s/nginx-https/container/nginx.conf.template

```diff
@@ -158,8 +158,8 @@ stream {
   # Enable logging when connections are being throttled.
   limit_conn_log_level notice;
   
-  # Allow 16 connections from the same IP address.
-  limit_conn two 16;
+  # Allow 64 connections from the same IP address.
```

Comment:
I think we will hit the issue of fairly early as MongoDB will keep ramping its connection count to 100 and NGINX will keep dropping them.
It is not a number of node issue now, it is the default MongoDB python driver connection pool size.
Ref:
http://api.mongodb.com/python/current/faq.html#how-does-connection-pooling-work-in-pymongo
https://stackoverflow.com/questions/11498734/best-practice-of-django-pymongo-pooling/11498804#11498804